index.qmd

```diff
@@ -4,8 +4,19 @@ title: "R-multiverse"
 
 <img src="logo/logo-hexbin.png" align="right" width="140" />
 
-A community-curated collection of R package releases, powered by [R-universe](https://r-universe.dev), available for installation from the repository at <https://community.r-multiverse.org>.
+#### A dual package repository system, powered by R-universe
 
-Anyone can contribute packages to the GitHub repository at <https://github.com/r-multiverse/contributions>. Once a package is registered, every new release is automatically built and checked with results available at <https://community.r-multiverse.org>. Package maintainers retain complete control over deployment.
+#### [Community](community.qmd)
 
-The R-multiverse project originated from the [R Consortium Repositories Working Group](https://github.com/RConsortium/r-repositories-wg) and is supported by an [R Consortium](https://www.r-consortium.org/) ISC grant.
+An open and inclusive R package repository with transparent governance standards, built on open source infrastructure.
+
+#### +
+
+#### [Production](production.qmd)
+
+A quarterly snapshot release of R packages, designed for deployment in production environments.
+
+<br />
```

Comment:
why not add this as a footer on all pages thus controlled by the Quarto config file? I think that's possible

Reply:
If you're talking about the R-consortium support, I did try and it ends up being too long and too ugly so I left it only on the homepage.

Reply:
@shikokuchuo just remembered we have such a footer on https://r-dbi.org/ Is it ugly :stuck_out_tongue_winking_eye: (it's short)

Reply:
No that looks good. So how about we just add "Supported by the R Consortium" in the footer? It was ugly when I tried to include the whole blurb about the working group...

Reply:
Yes it'd be a good idea I think, and you could keep the blurb in about.